Tribler/Core/TorrentChecker/torrent_checker.py

```diff
@@ -406,11 +422,8 @@ def _update_torrent_result(self, response):
         torrent_id = result[u'torrent_id']
         retries = result[u'tracker_check_retries']
 
-        # the result logic
-        is_good_result = seeders > 0 or leechers > 0
-
         # the status logic
-        if is_good_result:
+        if seeders > 0:
```

Comment:
My bad, I was thinking you guys meant UDP trackers. The DHT stuff used to simply do a get_peers which if peers were returned set the status to good. Should be pretty trivial to implement.